### PR TITLE
Blog: week 26 (June 27th – July 3rd)

### PR DESCRIPTION
**Pull request by jpopelka**

### CODE REVIEW

File: content/posts/weekly/June-2023.md [hunk 1]
```diff
@@ -58,3 +58,15 @@ weight: 51
...
+
+- The bug which prevented errors on Testing Farm request submission from being
+  correctly propagated to the user was fixed.
+  ([packit-service#2103](https://github.com/packit/packit-service/pull/2103))
+- `pull_from_upstream` jobs can now be retriggered with a comment
+  `/packit pull-from-upstream` in a dist-git pull request.
+  ([packit-service#2087](https://github.com/packit/packit-service/pull/2087))
+- Copr builds configured as a monorepo job will now be built in one Copr project together.
+  The bug with overwriting the statuses for monorepo jobs was fixed as well.
+  ([packit-service#2097](https://github.com/packit/packit-service/pull/2097))
```
Comment (lbarcziova) [inline content/posts/weekly/June-2023.md:72]:
```suggestion
```

Comment (lbarcziova) [reply content/posts/weekly/June-2023.md:72]: this one was already included last week

**Review by lbarcziova — COMMENTED** [review]
otherwise LGTM

File: content/posts/weekly/June-2023.md (hunk 1 above)
Comment (lachmanfrantisek) [reply content/posts/weekly/June-2023.md:72]: That's because I've moved the branches on Wednesday...

**Review by lbarcziova — APPROVED** [review]

**Comment by jpopelka** [comment]
regate

**Pull request merged**